apps/core/lib/core/schema/provider_scaffold.ex

```diff
@@ -18,6 +18,9 @@ defmodule Core.Schema.ProviderScaffold do
 
   defp provider_file(name, vsn) do
     Path.join([:code.priv_dir(:core), "scaffolds/terraform/providers/#{vsn}", "#{name}.eex"])
-    |> File.read!
+    |> case do
+       {:ok, resp} -> {:ok, resp} |> File.read!
```

Comment:
Path.join returns an ok tuple? also it's probably cleaner to write

```elixir
File.read!({:ok, resp})
```
Since it's inline like this and only one func call